controllers/volumereplicationgroup_controller.go

```diff
@@ -724,6 +773,30 @@ func (v *VRGInstance) processForDeletion() (ctrl.Result, error) {
 	return ctrl.Result{}, nil
 }
 
+// For now, for reginalDR (i.e. async mode enabled),
+// VolRep resources created by VRG have to be deletion
+// when VRG is deleted. For MetroDR (i.e. sync mode enabled)
+// nothing has to be done. So whether this VRG resource require
+// a requeue is a logical OR of whether async mode requires a
+// requeue and whether sync more requires a requeue.
+func (v *VRGInstance) deleteVRGHandleMode() bool {
+	asyncModeRequeue := false
+	syncModeRequeue := false
+
+	if v.instance.Spec.Async.Mode == ramendrv1alpha1.AsyncModeEnabled {
+		asyncModeRequeue = v.reconcileVRsForDeletion()
+	}
+
+	// for now nothing to do for MetroDR. VRG does not create VolRep or
+	// any other resource for MetroDR. Hence set syncModeRequeue to false
+	// indicating metroDR does not require a requeue.
+	if v.instance.Spec.Sync.Mode == ramendrv1alpha1.SyncModeEnabled {
+		syncModeRequeue = false
```

Comment:
syncModeRequeue is already set to false. Since Metro DR is NO OP in  deleteVRGHandleMode. This if block may not be required.